client/src/components/DataFiles/DataFilesModals/DataFilesAddProjectModal.jsx

```diff
@@ -82,6 +83,9 @@ const DataFilesAddProjectModal = () => {
       .min(3, 'Title must be at least 3 characters')
       .max(150, 'Title must be at most 150 characters')
       .required('Please enter a title.'),
+    description: Yup.string()
+      .max(800, 'Description must be at most 800 characters')
```

Comment:
where'd you get this max limit?  I'm actually working on updating descriptions in Digital Porous Media Portal (Digital Rocks) to be closer to DesignSafe, but since this isn't currently a "publishing" portal codebase (yet), I'd suggest using min:50 and max:5000 like we do for non-primary descriptions in DesignSafe.  our primary (top level) descriptions in DesignSafe and DPMP are or will be soon min:1000, but that seems like a lot to require right now in Core.

Reply:
[The existing shared workspace description maximum is 800 characters.](https://github.com/TACC/Core-Portal/blob/task/WP-1099/client/src/components/DataFiles/DataFilesModals/tests/DataFilesProjectEditDescriptionModal.test.jsx#L122) I'd be willing to change it to 5000, like in DS, but it might warrant an additional discussion?